tests/conftest.py

```diff
@@ -16,3 +17,8 @@ def loguru_caplog(caplog):
     handler_id = logger.add(caplog.handler, format="{message}")
     yield caplog
     logger.remove(handler_id)
+
+
+def create_citext_extension(engine: Engine) -> None:
```

Comment:
This is required for the test db to work properly. Without it CI fails because the `citext` extension is missing from the test db

Reply:
nice one, @TheAndrewJackson